src/Utilities/DeleteModal.test.js

```diff
@@ -1,48 +1,33 @@
 /* eslint-disable camelcase */
 import React from 'react';
-import { mount, shallow } from 'enzyme';
-import toJson from 'enzyme-to-json';
 import DeleteModal from './DeleteModal';
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
 
 describe('EntityTable', () => {
   describe('DOM', () => {
-    it('should render correctly', () => {
-      const wrapper = shallow(<DeleteModal />);
-      expect(toJson(wrapper)).toMatchSnapshot();
-    });
+    it('should render correctly with no system', () => {
```

Comment:
The snapshot for this test doesn't really tell anything anymore.[0]
In this case, it should be safe to just remove it.
If in case of no systems the DeleteModal would/should render a warning we could query that, but that is not the case here. 


[0] https://github.com/RedHatInsights/insights-inventory-frontend/pull/2033/files#diff-64d05750a711d29ab9558378f56ac984e52c92e258eafb6402ccdf0fa4366934R204-R207